### ISSUE TITLE
Add unit tests

### ISSUE THREAD

**Issue by emdupre** [ME-ICA/tedana#15]
_From @emdupre on November 15, 2017 16:35_

We have one larger integration test in `test_tedana.py`, but each function should also have its own unit test. This will be significantly easier once individual functions have been created as in #4.

_Copied from original issue: emdupre/tedana#6_

**Comment by tsalo** [comment]
I think this issue can be closed, given that it has been superseded by #69 and that we have agreed to rely on unit tests (rather than integration tests) for measuring coverage.

**Comment by emdupre** [comment]
I had thought we were going to add one more to `tedana` ! Just the five echo test case, and we'd run the two (three and five echos) in parallel. WDYT ?
